packages/console/src/components/Checkbox/index.tsx

```diff
@@ -1,23 +1,36 @@
 import { nanoid } from 'nanoid';
-import React, { forwardRef, InputHTMLAttributes, ReactNode, Ref, useState } from 'react';
+import React, { ReactNode, useState } from 'react';
 
 import Icon from './Icon';
 import * as styles from './index.module.scss';
 
-type Props = Omit<InputHTMLAttributes<HTMLInputElement>, 'type'> & {
+type Props = {
+  // eslint-disable-next-line react/boolean-prop-naming
+  value: boolean;
+  onChange: (value: boolean) => void;
   label?: ReactNode;
+  // eslint-disable-next-line react/boolean-prop-naming
+  disabled: boolean;
```

Comment:
Do we use disabled across the project? Or isDisabled?

Reply:
`disabled` is commonly used, because native html element (such as input) use `disabled`